web-platform/website/src/components/AuthenticatedPage.tsx

```diff
@@ -22,7 +25,8 @@ export function AuthenticatedPage(props: {
 
   if (
     router.pathname !== "/privacy-policy" &&
-    (!userDocument || !userDocument.enrolled)
+    (!userDocument || !userDocument.enrolled) &&
+    !isFlagActive(Flags.onboardingV2)
```

Comment:
We can just check for flag within /privacy-policy and all these files can be reverted. Then depending on whether v2 flag is true or not we can decide to show modal or inline privacy-policy?

Reply:
I did that originally in /privacy-policy but this code bypasses it... Also, once this new onboarding flow launches we will have to remove the privacy policy and profile pages from the flow per the designs.